models/order/src/order-from-cart-draft/presets/sample-data-fashion/john-cart-02.ts

```diff
@@ -0,0 +1,15 @@
+import { KeyReference } from '@commercetools-test-data/commons';
+import * as OrderFromCartDraft from '../../';
+import { constants } from '../../..';
+import { TOrderFromCartDraftBuilder } from '../../../types';
+
+const johnCart02 = (): TOrderFromCartDraftBuilder =>
+  OrderFromCartDraft.presets
+    .empty()
+    .version(null!)
+    .cart(KeyReference.random().key('john-02-cart').typeId('cart'))
```

Comment:
A note above these lines in the presets to use the key from the cart model presets when available might be nice

Reply:
Those are available now!